### ISSUE TITLE
Doc/Gradle: deprecated documentation on proxy usage

### ISSUE THREAD

**Issue by Lucas-C** [dependency-check/DependencyCheck#3179]
**Describe the bug**
When calling `gradle dependencyCheckAnalyze` I get this warning:
```
Deprecated configuration `proxy { server='... }`; please update your configuration to use the gradle proxy configuration
```

However the latest version of the documentation describe this deprecated syntax:
https://jeremylong.github.io/DependencyCheck/dependency-check-gradle/configuration.html

**Version of dependency-check used**
6.1.1

**To Reproduce**
My `build.gradle` contains this:
```groovy
dependencyCheck {
...
        proxy {
            server = "$System.env.PROXY_HOST"
            port = "$System.env.PROXY_PORT" as Integer
            nonProxyHosts = "$System.env.NO_PROXY_HOSTS".split('|')
```

**Expected behavior**
No warning, or a documentation that describe how to configure the _gradle proxy configuration_

**Additional context**
The warning comes from this code line:
https://github.com/jeremylong/dependency-check-gradle/blob/main/src/main/groovy/org/owasp/dependencycheck/gradle/tasks/ConfiguredTask.groovy#L173

**Comment by jeremylong** [comment]
Documentation has been updated.

**Comment by Lucas-C** [comment]
Thank you !
